tests/Unit/CronTest.php

```
@@ -0,0 +1,313 @@
+			array(
+				'return' => function ( $key ) {
+					unset( $this->transients[ $key ] );
+					$this->delete_transient_calls[] = $key;
+					return true;
+				},
+			)
+		);
+
+		// Cron jobs that touch the DB / make no-op calls when no work is found.
+		WP_Mock::userFunction( 'get_posts', array( 'return' => array() ) );
+		WP_Mock::userFunction( 'apply_filters', array( 'return_arg' => 1 ) );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\WP_Mock' in method 'setUp'. (undefined)

(StaticAccess)